src/main/java/it/gov/pagopa/reward/notification/repository/RewardsNotificationRepositoryExtended.java

```diff
@@ -15,4 +17,7 @@ public interface RewardsNotificationRepositoryExtended {
     Mono<String> updateExportStatus(String rewardNotificationId, String iban, String checkIbanResult, String exportId);
 
     Mono<RewardsNotification> saveIfNotExists(RewardsNotification rewardsNotification);
+
+    Flux<RewardsNotification> findAllWithFilters(String organizationId, String initiativeId, String exportId, Pageable pageable, ExportDetailFilter filters);
```

Comment:
redundant name.
let's keep Pageable always as last

fix invocations to respect new order
```suggestion
    Flux<RewardsNotification> findAll
    WithFilters(String organizationId, String initiativeId, String exportId, ExportDetailFilter filters, Pageable pageable);
```